client/app/scripts/components/node-details/__tests__/node-details-table-test.js

```diff
@@ -125,4 +138,23 @@ describe('NodeDetailsTable', () => {
       matchColumnValues('kubernetes_namespace', ['00000', '1111', '12', '5']);
     });
   });
+
+  describe('uptime duration', () => {
+    it('sorts by column', () => {
+      component = TestUtils.renderIntoDocument((
+        <Provider store={configureStore()}>
+          <NodeDetailsTable
+            columns={columns}
+            sortedBy="uptime"
+            nodeIdKey="id"
+            nodes={nodes}
+          />
+        </Provider>
+      ));
+
+      matchColumnValues('uptime', ['1 second', '2 seconds', '4 seconds', '30 seconds']);
+      clickColumn('Uptime');
+      matchColumnValues('uptime', ['30 seconds', '4 seconds', '2 seconds', '1 second']);
```

Comment:
Would be nice to test against longer durations as well, so that more units are involved than just _seconds_ :)